src/ck3/tables/modifs.rs

```diff
@@ -29,12 +40,10 @@ pub fn lookup_modif(name: &Token, data: &Everything, warn: Option<Severity>) ->
         "_ai_sociability",
         "_ai_vengefulness",
         "_ai_zeal",
-        "_different_culture_opinion",
-        "_different_faith_opinion",
-        "_same_culture_opinion",
-        "_same_faith_opinion",
     ] {
         if let Some(s) = name.as_str().strip_suffix(sfx) {
+            // Verify modifier format
+            data.verify_exists(Item::ModifierFormat, name);
```

Comment:
I checked my memory and validate_modif already does this

https://github.com/amtep/ck3-tiger/blob/d93f8da63053dbdad126f95f825e3b812f637cd8/src/modif.rs#L105